### DISCUSSION TITLE
LabVIEW 2019 for Next Major Release - any objections or considerations?

### DISCUSSION THREAD

**Discussion by jimkring** [vipm-io/vi-tester#71] [Polls]
We're pushing for a new release and I'm wondering what LabVIEW version we should use.

I'm tempted to go with LabVIEW 2019, but I'm wondering if that would cause problems for core users and developers.

Any thoughts?

@JamesMc86 @samsharp99 @kosist @omarmussa

**Comment by samsharp99** [comment]
Hi Jim,

We have quite a few projects still in LabVIEW 2017 but slowly transitioning
to 2021.

So 2017 would be preferred.

Regards,
Sam

**Comment by jimkring** [comment]
Sure, Sam.  I can go with LV2017 pretty easily myself. If that would help
you, we can stick with that.

On Sat, Aug 27, 2022 at 11:41 AM Sam Sharp ***@***.***> wrote:

> Hi Jim,
>
> We have quite a few projects still in LabVIEW 2017 but slowly transitioning
> to 2021.
>
> So 2017 would be preferred.
>
> Regards,
> Sam
>
> On Sat, 27 Aug 2022, 18:08 Jim Kring, ***@***.***> wrote:
>
> > We're pushing for a new release and I'm wondering what LabVIEW version we
> > should use.
> >
> > I'm tempted to go with LabVIEW 2019, but I'm wondering if that would
> cause
> > problems for core users and developers.
> >
> > Any thoughts?
> >
> > @JamesMc86 <https://github.com/JamesMc86> @samsharp99
> > <https://github.com/samsharp99> @kosist <https://github.com/kosist>
> > @omarmussa <https://github.com/omarmussa>
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/vipm-io/vi-tester/discussions/71>, or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ABY7WORARVC7NSZNSFLAHFDV3JDSBANCNFSM57ZTP3TQ
> >
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vipm-io/vi-tester/discussions/71#discussioncomment-3490558>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAC5D6GOLHK2SGXSONVYSCLV3JOOTANCNFSM57ZTP3TQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Jim Sent from my mobile.

**Comment by jimkring** [comment]
Side note: i’d really love to start using LabVIEW 2020 or greater for
community projects, simply because there is a Community Edition and we can
open up to a wider audience of contributors.

On Sat, Aug 27, 2022 at 11:50 AM Jim Kring ***@***.***> wrote:

> Sure, Sam.  I can go with LV2017 pretty easily myself. If that would help
> you, we can stick with that.
>
> On Sat, Aug 27, 2022 at 11:41 AM Sam Sharp ***@***.***>
> wrote:
>
>> Hi Jim,
>>
>> We have quite a few projects still in LabVIEW 2017 but slowly
>> transitioning
>> to 2021.
>>
>> So 2017 would be preferred.
>>
>> Regards,
>> Sam
>>
>> On Sat, 27 Aug 2022, 18:08 Jim Kring, ***@***.***> wrote:
>>
>> > We're pushing for a new release and I'm wondering what LabVIEW version
>> we
>> > should use.
>> >
>> > I'm tempted to go with LabVIEW 2019, but I'm wondering if that would
>> cause
>> > problems for core users and developers.
>> >
>> > Any thoughts?
>> >
>> > @JamesMc86 <https://github.com/JamesMc86> @samsharp99
>> > <https://github.com/samsharp99> @kosist <https://github.com/kosist>
>> > @omarmussa <https://github.com/omarmussa>
>> >
>> > —
>> > Reply to this email directly, view it on GitHub
>> > <https://github.com/vipm-io/vi-tester/discussions/71>, or unsubscribe
>> > <
>> https://github.com/notifications/unsubscribe-auth/ABY7WORARVC7NSZNSFLAHFDV3JDSBANCNFSM57ZTP3TQ
>> >
>> > .
>> > You are receiving this because you were mentioned.Message ID:
>> > ***@***.***>
>> >
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/vipm-io/vi-tester/discussions/71#discussioncomment-3490558>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AAC5D6GOLHK2SGXSONVYSCLV3JOOTANCNFSM57ZTP3TQ>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
> --
> Jim Sent from my mobile.
>
-- 
Jim Sent from my mobile.

**Comment by omarmussa** [comment]
I’m good with >= 2019 or >= 2020 but need at least 2020 to be supported ideally.

Thanks,
Omar

Written on my mobile phone, please excuse any typos.

**Comment by kosist** [comment]
I would vote for 2017, but any other newer version is also good, especially LV 2020 and greater (due to Community Edition).

Thanks, 
Ivan.

**Comment by matthiasbaudot** [comment]
Almost all our codebase and reuseable libraries are LabVIEW 2016. We're planning to migrate all our unit tests to Caraya, but at the moment, it's all VI Tester.
